hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanner.java

```diff
@@ -472,4 +499,21 @@ private boolean isFileSliceNeededForPendingCompaction(FileSlice fileSlice) {
   private boolean isFileGroupInPendingCompaction(HoodieFileGroup fg) {
     return fgIdToPendingCompactionOperations.containsKey(fg.getFileGroupId());
   }
+
+  public boolean isDeletePartitionOperation(Option<HoodieInstant> instantToRetain) {
```

Comment:
can we make it private if not intend to use else where? also prefer to have a static helper for this, instead of couple it with CleanPlanner

Reply:
it actually looks better to be in some timeline utils class instead of in `CleanPlanner`